platform/kube/controller.go

```diff
@@ -43,11 +43,12 @@ const (
 
 // ControllerOptions stores the configurable attributes of a Controller.
 type ControllerOptions struct {
-	// Namespace to restrict controller to (empty to disable restriction)
-	Namespace    string
-	AppNamespace string
-	ResyncPeriod time.Duration
-	DomainSuffix string
+	// Namespace in which the controller runs. Can be "" if the controller does not run in a namespace
+	Namespace string
+	// Namespace the controller watches. If set to meta_v1.NamespaceAll (""), controller watches all namespaces
```

Comment:
Thank you !

Reply:
Why did you change the meaning of Namespace field? The previous semantics was WatchedNamespace.